x/concentrated-liquidity/lp_test.go

```diff
@@ -851,53 +840,48 @@ func (s *KeeperTestSuite) TestAddToPosition() {
 			amount0ToAdd:       amount0PerfectRatio,
 			amount1ToAdd:       amount1PerfectRatio,
 		},
-		"error: attempt to add negative asset0 to position": {
+		"error: not position owner": {
 			// setup parameters for creating a pool and position.
-			setupConfig: baseCase,
+			setupConfig:    baseCase,
+			senderNotOwner: true,
 
 			// system under test parameters
 			sutConfigOverwrite: &lpTest{
-				expectedError: types.NegativeAmountAddedError{PositionId: 1, Asset0Amount: amount0PerfectRatio.Neg(), Asset1Amount: amount1PerfectRatio},
+				expectedError: types.NotPositionOwnerError{PositionId: 1, Address: invalidSender.String()},
 			},
-			lastPositionInPool: true,
-			timeElapsed:        defaultTimeElapsed,
-			amount0ToAdd:       amount0PerfectRatio.Neg(),
-			amount1ToAdd:       amount1PerfectRatio,
+			timeElapsed:  defaultTimeElapsed,
+			amount0ToAdd: amount0PerfectRatio,
+			amount1ToAdd: amount1PerfectRatio,
 		},
-		"error: attempt to add negative asset1 to position": {
+		"error: minimum amount 0 is less than actual amount": {
 			// setup parameters for creating a pool and position.
 			setupConfig: baseCase,
 
 			// system under test parameters
 			sutConfigOverwrite: &lpTest{
-				expectedError: types.NegativeAmountAddedError{PositionId: 1, Asset0Amount: amount0PerfectRatio, Asset1Amount: amount1PerfectRatio.Neg()},
+				amount0Minimum: sdk.NewInt(1997960),
+				expectedError: types.InsufficientLiquidityCreatedError{
+					Actual:      sdk.NewInt(1997954),
+					Minimum:     sdk.NewInt(1997960),
+					IsTokenZero: true,
+				},
 			},
-			lastPositionInPool: true,
-			timeElapsed:        defaultTimeElapsed,
-			amount0ToAdd:       amount0PerfectRatio,
-			amount1ToAdd:       amount1PerfectRatio.Neg(),
+			timeElapsed:  defaultTimeElapsed,
+			amount0ToAdd: amount0PerfectRatio,
+			amount1ToAdd: amount1PerfectRatio,
 		},
-		"error: attempt to add negative amounts for both assets to position": {
```

Comment:
I think we still need tests for:
- negative amt0
- negative amt1
- zero for both amt0 and amt1

Reply:
Good call on this, the original reason I took these out was because it would error when preparing environment for the testcase, but found a way out and added in